### PR TITLE
PR Featurebranch/sprint3 to Dev

### PR DESCRIPTION
**Pull request by padmanabhamsaikeertana**

### CODE REVIEW

File: customer_service/src/main/java/com/example/customer_service/mapper/CustomerMapper.java
Comment (sreejanomula) [inline customer_service/src/main/java/com/example/customer_service/mapper/CustomerMapper.java:1]: when i compared it i saw that mapper class  is deleted

File: customer_service/src/test/java/com/example/customer_service/mapper/CustomerServiceMapperTest.java
Comment (sreejanomula) [inline customer_service/src/test/java/com/example/customer_service/mapper/CustomerServiceMapperTest.java:1]: when i compared it i saw that mappertest class is deleted

File: customer_service/src/main/java/com/example/customer_service/mapper/CustomerMapper.java
Comment (padmanabhamsaikeertana) [reply customer_service/src/main/java/com/example/customer_service/mapper/CustomerMapper.java:1]: I have removed my mapper class and created a new mapper interface

File: customer_service/src/test/java/com/example/customer_service/mapper/CustomerServiceMapperTest.java
Comment (padmanabhamsaikeertana) [reply customer_service/src/test/java/com/example/customer_service/mapper/CustomerServiceMapperTest.java:1]: I have removed my mapper class test and created a new mapper interface test

**Review by sreejanomula — CHANGES_REQUESTED** [review]
can you update the changes

**Review by sreejanomula — APPROVED** [review]

**Pull request merged**